dot/network/test_helpers.go

```diff
@@ -190,12 +194,16 @@ var starting, _ = variadic.NewUint64OrHash(uint64(1))
 
 var one = uint32(1)
 
-var testBlockRequestMessage = &BlockRequestMessage{
-	RequestedData: RequestedDataHeader + RequestedDataBody + RequestedDataJustification,
-	StartingBlock: *starting,
-	EndBlockHash:  &common.Hash{},
-	Direction:     1,
-	Max:           &one,
+func NewTestBlockRequestMessage(t *testing.T) *BlockRequestMessage {
+	t.Helper()
+
+	return &BlockRequestMessage{
+		RequestedData: RequestedDataHeader + RequestedDataBody + RequestedDataJustification,
+		StartingBlock: *starting,
+		EndBlockHash:  &common.Hash{},
+		Direction:     1,
+		Max:           &one,
```

Comment:
```suggestion
		Max:           uint32Ptr(1),
```

and define

```go
func uint32Ptr(n uint32) *uint32 { return &n }
```

So we avoid relying on global vars, especially for global vars pointed to